### ISSUE TITLE
Uncaught Error: Problem reading log4js config { appenders: [ { type: 'stdout' } ], replaceConsole...

### ISSUE THREAD

**Issue by prefonty** [facebooknuclide/hyperclick#27]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.4
**Thrown From**: [hyperclick](https://github.com/facebooknuclide/hyperclick) package 0.1.2

### Stack Trace

Uncaught Error: Problem reading log4js config { appenders: [ { type: 'stdout' } ], replaceConsole: false }. Error was "Cannot find module "."" (Error: Cannot find module "."
    at /Users/Yanik/.atom/packages/hyperclick/index.js:1:167342
    at E (/Users/Yanik/.atom/packages/hyperclick/index.js:1:167412)
    at T (/Users/Yanik/.atom/packages/hyperclick/index.js:1:167448)
    at /Users/Yanik/.atom/packages/hyperclick/index.js:1:165747
    at Array.forEach (native)
    at u (/Users/Yanik/.atom/packages/hyperclick/index.js:1:165727)
    at f (/Users/Yanik/.atom/packages/hyperclick/index.js:1:166205)
    at x (/Users/Yanik/.atom/packages/hyperclick/index.js:1:166995)
    at Object.<anonymous> (/Users/Yanik/.atom/packages/hyperclick/index.js:1:169206)
    at t (/Users/Yanik/.atom/packages/hyperclick/index.js:1:194)
    at a (/Users/Yanik/.atom/packages/hyperclick/index.js:1:156157)
    at _getMousePositionAsBufferPosition (/Users/Yanik/.atom/packages/hyperclick/index.js:1:161350)
    at _onMouseMove (/Users/Yanik/.atom/packages/hyperclick/index.js:1:159196))

```
At /Users/Yanik/.atom/packages/hyperclick/index.js:1

Error: Problem reading log4js config { appenders: [ { type: 'stdout' } ], replaceConsole: false }. Error was "Cannot find module "."" (Error: Cannot find module "."
    at /packages/hyperclick/index.js:1:167342
    at /packages/hyperclick/index.js:1:167412)
    at /packages/hyperclick/index.js:1:167448)
    at /packages/hyperclick/index.js:1:165747
    at Array.forEach (native)
    at /packages/hyperclick/index.js:1:165727)
    at /packages/hyperclick/index.js:1:166205)
    at /packages/hyperclick/index.js:1:166995)
    at /packages/hyperclick/index.js:1:169206)
    at /packages/hyperclick/index.js:1:194)
    at /packages/hyperclick/index.js:1:156157)
    at _getMousePositionAsBufferPosition (/packages/hyperclick/index.js:1:161350)
    at _onMouseMove (/packages/hyperclick/index.js:1:159196))
    at /packages/hyperclick/index.js:1:166259)
    at /packages/hyperclick/index.js:1:166995)
    at /packages/hyperclick/index.js:1:169206)
    at /packages/hyperclick/index.js:1:194)
    at /packages/hyperclick/index.js:1:156157)
    at _getMousePositionAsBufferPosition (/packages/hyperclick/index.js:1:161350)
    at _onMouseMove (/packages/hyperclick/index.js:1:159196)
```

### Commands

```
  2x -3:18.8.0 emmet:expand-abbreviation-with-tab (input.hidden-input)
     -3:12.3.0 core:save (input.hidden-input)
  5x -1:45.1.0 core:backspace (input.hidden-input)
     -1:34.9.0 core:save (input.hidden-input)
     -0:45.2.0 tree-view:reveal-active-file (div.title)
     -0:40.5.0 tree-view:remove (span.name.icon.icon-file-text)
     -0:22 tree-view:reveal-active-file (div.title)
```

### Non-Core Packages

```
atom-beautify 0.30.3 
atom-css-clean 2.20.5 
atom-css-comb 3.2.0 
atom-html-preview 0.1.22 
atom-wrap-in-tag 0.6.0 
auto-indent 0.5.0 
autocomplete-html-entities 0.1.0 
build 0.68.0 
busy 0.7.0 
busy-signal 1.4.3 
color-picker 2.2.5 
compare-files 0.8.0 
css-declaration-sorter 1.0.1 
double-tag 0.9.0 
drag-relative-path 1.1.2 
emmet 2.4.3 
expand-region 0.3.0 
fonts 3.0.2 
git-blame 1.1.1 
git-plus 7.9.3 
goto-definition 1.3.4 
html-include-resource 1.3.0 
hyperclick 0.1.2 
indent-guide-improved 1.4.12 
intentions 1.1.2 
jquery-snippets 11.0.0 
linter 2.2.0 
linter-stylelint 3.10.0 
linter-ui-default 1.6.2 
minimap 4.28.2 
minimap-pigments 0.2.2 
monokai 0.24.0 
open-in-browser 0.5.2 
php-integrator-autocomplete-plus 1.5.0 
php-integrator-base 2.1.12 
php-integrator-call-tips 1.2.1 
php-integrator-linter 1.3.0 
php-integrator-navigation 1.1.2 
php-integrator-refactoring 1.3.1 
php-integrator-tooltips 1.1.5 
pigments 0.39.1 
project-manager 3.3.5 
remote-sync 4.1.5 
sass-autocompile 0.13.3 
sort-lines 0.14.0 
split-diff 1.4.3 
sync-settings 0.8.2 
tag 0.5.0 
terminal-plus 0.14.5 
terminal-status 1.6.8 
trailing-spaces 0.4.0 
tree-view-copy-relative-path 1.2.0 
web-view 3.0.0 
```

**Comment by hansonw** [comment]
Thanks! Fixed in f42a85c8aa7b2ddc26b7b22d346bd10ead5eac28, I'll put out a new release shortly.

**Comment by hansonw** [comment]
Fixed in v0.1.3
